### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Notebooks - Create_Keyboard: The tooltip info is not available for the buttons such as 'Add cell', 'Split', 'Delete', 'Add cell up', 'Add cell down', 'Bold', 'Italic' etcetera for the '+code' and '+text' buttons using keyboard tab keys unlike mouse.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19875]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev3;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#Keyboard;#AINotInScope;#WCAG1.1.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Tab to 'File' menu.
5. Tab to 'New Notebook' and select.
6. Tab to '+Code' button and select.
8. Use keyboard to access buttons associated and observe the tooltip info.

## Actual:
The tooltip info is not available for the buttons such as 'Add cell', 'Split', 'Delete', 'Add cell up', 'Add cell down', 'Bold', 'Italic' etcetera for the '+code' and '+text' buttons using keyboard tab keys unlike mouse.

### Same issue can be observed for the following scenario:

1. Dashboard (Tooltip not available for ‘<<’ hide text labels, ‘^’ collapse widget, ‘...’ show more widgets using keyboard)
2. Terminal_New Terminal (Tooltip not available for new terminal button using keyboard)
3. Browse Preview_Azure tree item (Tooltip not appearing for ‘Refresh’ and ‘Sign in’ button)

## Expected:
The tooltip info should be available for the buttons such as 'Add cell', 'Split', 'Delete', 'Add cell up', 'Add cell down', 'Bold', 'Italic' etcetera for the '+code' and '+text' buttons using keyboard tab keys like mouse. The tooltip should be visible on tab key focus on the button and on using escape to close the tooltip should get dismissed, but focus should stay on the button

## User Impact:
Keyboard users will face difficulty in accessing the important tooltip information if not available and might not be able to recognize a particular control especially in absence of visual label.

## Attachment:
![13_Tooltip](https://user-images.githubusercontent.com/98393966/176432706-3d34ac8c-9b6e-44e0-9b54-031a1e00f850.gif)

**Comment by alanrenmsft** [comment]
@bbarik06 this is an external bug: https://github.com/microsoft/vscode/issues/132344
Just like other similar issues, could you please downgrade this one to sev3?

**Comment by alanrenmsft** [comment]
dupe of #19566

**Comment by sspatil123** [comment]
GitHubTags:#A11ySev3
